xrpl/models/requests/get_aggregate_price.py

```diff
@@ -0,0 +1,60 @@
+"""
+This module defines the GetAggregatePrice request API. It is used to fetch aggregate
+statistics about the specified PriceOracles
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional
+
+from typing_extensions import TypedDict
+
+from xrpl.models.requests.request import Request, RequestMethod
+from xrpl.models.required import REQUIRED
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class GetAggregatePrice(Request):
```

Comment:
Same here. The docstring should match the documentation:
https://opensource.ripple.com/docs/xls-47d-price-oracles/public-api-methods/get_aggregate_price/

Reply:
fixed in [3957867](https://github.com/XRPLF/xrpl-py/pull/701/commits/3957867b12e9b76120b97bffd6a9d8d1b64f3cfa)